### DISCUSSION TITLE
Show only the Docs story

### DISCUSSION THREAD

**Discussion by jonathanadler** [storybookjs/storybook#29425] [Help] [answered]
### Summary

My components have several stories, some are for documentation, and some for testing (chromatic).

I use the `autodocs` tag for some of my stories, and I was wondering if there's a way to hide stories that are added to the Docs page from the sidebar, since I don't need the duplication.
See example in image:

![Screenshot 2024-10-22 at 16 14 13](https://github.com/user-attachments/assets/a2cf9d51-b3b0-4311-8bc8-6cf4a3c15b31)

I searched for an existing issue/discussion, but couldn't find anything other than [this](https://github.com/storybookjs/storybook/discussions/24849)

### Additional information

I'm using `storybook@^8.3.6`

**Comment by smlimasu** [comment, marked as answer]
Set !dev in story tags, see https://storybook.js.org/docs/writing-stories/tags

**Reply by jonathanadler** [reply]
I totally missed that, thanks it worked!
